pages/glossary/browsing-context-related-documents.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Set of browsing context related documents

```

Comment:
I think we can make this a lot easier to understand if we say something like "on the same web page". I suggest you expand this to not just be about documents, but about nodes, which either have the same owned document, or are owned by two document with the same top-level browser context. That'll make the definition easier to use.

Reply:
The problem being that "web page" is less well defined that "browsing context", as you pointed in #869 

I agree that nodes will be better for that def.

Reply:
I tried to use node inside the definition. I am not completely sure this is better since browsing context is ultimately a document concept more than a node concept, so this creates an extra indirection (node => document => browsing context).

Maybe it could be done by still having "browsing context related **documents**" and use it to define "browsing context related **nodes**" (as all nodes inside browsing context related documents). What do you think?